src/app/components/drawer/DrawerRail.js

```diff
@@ -54,7 +54,7 @@ const messages = defineMessages({
 
 const DrawerRail = (props) => {
   const testPath = window.location.pathname;
-  const isSettingsPage = /\/settings\/[a-zA-Z0-9]+/.test(testPath);
+  const isSettingsPage = /^\/[^/]+\/settings?($|\/)/.test(testPath);
```

Comment:
What's this regular expression supposed to do? When I run something like

```
"https://checkmedia.org/check-testing/settings/workspace".match(/^\/[^/]+\/settings?($|\/)/) 
```

I get no match, which leads me to believe this doesn't work for testing if we are on the settings page at all. It looks like the regex is actually searching for `/` and then at least one of something that's not `/` and then `/` and then `settings` and then either the end of a string or another `/`. See here:

https://regex101.com/r/g80Mip/1

Maybe you meant something like `/[^/]+\/settings?/`? I'm just not sure what's going on here and it doesn't seem to match the trivial string of a settings url.

Reply:
@dariusk I was just trying to figure out why this link was behaving differently than the others, so I copied the shared feeds line from that same file:
`const isFeedPage = /^\/[^/]+\/feed(s)?($|\/)/.test(testPath);`
but perhaps that was overthinking/underthinking it.

I replaced it with your suggestion and it seems to work, take a look again